standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -167,6 +167,39 @@ const string HIVE_FILTER_FIELD_LAST_ACCESS = "hive_filter_field_last_access__"
 const string HIVE_FILTER_FIELD_TABLE_NAME = "hive_filter_field_tableName__"
 const string HIVE_FILTER_FIELD_TABLE_TYPE = "hive_filter_field_tableType__"
 
+/** Table maintenance operation type. */
+enum MaintenanceOpType {
+    COMPACTION = 1,
+    SNAPSHOT_EXPIRY = 2,
+    STATS_REBUILD= 3,
+    MV_BUILD= 4,
```

Comment:
What's the `MaintenanceOpType` and `MaintenanceOpStatus` for?

Reply:
These are intended as descriptors for data life-cycle management features.

Reply:
Removed them from Thrift def.